.husky/pre-push

```diff
@@ -0,0 +1 @@
+npm run test:unit || (echo '\n (⊙＿⊙) Your forgot to run unit tests!\n'; false)
```

Comment:
Maybe later if tests are going to take a lot of time we can think if pre-push is really needed as it will hinder the movement. But for now it took less than 800ms.